examples/indented_tree.py

```diff
@@ -3,28 +3,34 @@
 ===================
 
 A demonstration of parsing indentation (“whitespace significant” language)
-and the usage of the Indenter class.
+and the usage of the `Indenter` class.
 
 Since indentation is context-sensitive, a postlex stage is introduced to
-manufacture INDENT/DEDENT tokens.
+manufacture `INDENT`/`DEDENT` tokens.
 
-It is crucial for the indenter that the NL_type matches
-the spaces (and tabs) after the newline.
+It is crucial for the indenter that the `NL_type` matches the spaces (and
+tabs) after the newline.
+
+If your whitespace-significant grammar supports comments, then `NL_type`
+must match those comments too. Otherwise, comments that appear in the middle
+of a line will [confuse Lark][1].
```

Comment:
Hmm, I wonder if this is supposed to use RST syntax instead of Markdown. If I can get some guidance on how to build and review the docs locally, I can confirm the correct syntax to use.

Reply:
The docs are fairly easy to build imho. Just `cd docs && make` (but make sure to install all the dependencies through `docs/requirements.txt`)